source/declarations.tex

```diff
@@ -406,7 +406,8 @@
 \indextext{restriction!\idxcode{extern}}%
 The \tcode{extern} specifier shall be applied only to the declaration of a variable
 or function. The \tcode{extern} specifier shall not be used in the
-declaration of a class member or function parameter.
+declaration of a class member, function parameter, or block-scope variable with
+an \grammarterm{initializer}.
```

Comment:
This change makes it sound like it's OK to use `extern` with a class member or function parameter so long as there's no initializer. Moving the "block-scope variable with an initializer" case to the start of the list would help.